src/Resources/config/services/services.xml

```diff
@@ -157,7 +157,9 @@
         />
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Api\FindologicSearchService" autowire="true" />
-        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true" />
+        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true">
+            <argument type="string" key="$shopwareVersion">%kernel.shopware_version%</argument>
```

Comment:
Should there be a leading `$` for the key?

`key="$shopwareVersion"`

Reply:
Yes:
https://symfony.com/doc/current/service_container.html#manually-wiring-arguments

Reply:
Discussed in Slack, resolved.